docs/operate/get-started/other-hardware/dependencies.md

```diff
@@ -172,46 +179,52 @@ If you need to maintain the state of your resource, see [(Optional) Create and e
 
 ### Optional dependencies
 
+If an optional dependency does not start, the module will continue to build and reconfigure without it.
```

Comment:
```suggestion
If an optional dependency does not start, the modular resource will continue to build and reconfigure without it.
```
[nit] Modules don't build and reconfigure, modular resources do.

Reply:
Oops yes ty for catching